.code-samples.meilisearch.yaml

```
@@ -80,7 +80,13 @@ delete_an_index_1: |-
 get_one_document_1: |-
   await client.Index("movies").GetDocumentAsync<Movie>(25684, new List<string> { "id", "title", "poster", "release_date" });
 get_documents_1: |-
-  await client.Index("movies").GetDocumentsAsync<Movie>(new DocumentsQuery() { Limit = 2 });
+  await client.Index("movies").GetDocumentsAsync<Movie>(new DocumentsQuery() { Limit = 2, Filter = "genres = action" });
+get_documents_post_1: |-
+  await client.Index("movies").GetDocumentsAsync<Movie>(new DocumentsQuery() { 
+      Limit = 3, 
+      Fields = new List<string> { "title", "genres", "rating", "language"}, 
+      Filter = "genres = action OR genres = adventure" 
```

```suggestion
      Filter = "(rating > 3 AND (genres=Adventure OR genres=Fiction)) AND language=English" 
```
There is a mistake in the issue sorry about that